### ISSUE TITLE
Waterline 0.10.22 breaks defaultsTo

### ISSUE THREAD

**Issue by JasonMore** [balderdashy/waterline#1032]
In our app we used to be able to count on "defaultsTo" to fill in a boolean value on our model when creating it. Like so:

Message.js
```
module.exports = {
	attributes: {
		text: {
			type: 'string',
			required: true,
			minLength: 1
		},
		edited: {
			type: 'boolean',
			required: false,
			defaultsTo: false
		}
	}
};
```

MessageController.js
```
Message.create({
		text: text
	}).then(function (message) {
		// stuff...
	});
```

However, after the upgrade to waterline 0.10.22, we now get this validation error:
```
Invalid attributes sent to Message:
 • edited
   • "required" validation rule failed for input: false
```

Adding the value in fixes it, but I'm assuming this is a breaking change for waterline.
```
Message.create({
		text: text
		edited: false
	}).then(function (message) {
		// stuff...
	});
```

**Comment by JasonMore** [comment]
Sorry guys, ended up being the ` required: false, ` that was messing it up. Weird

**Comment by dmarcelino** [comment]
Hey @JasonMore, ` required: false,` should be fixed by PR #982 which was released in waterline v0.10.22 a few days ago.
